.env-github-actions

```diff
@@ -7,12 +7,14 @@ AML_SECRET_KEY=amlsecretkey
 AML_LOCATION_PROVIDER=http://ip2country:5000/{}
 AML_DEBUG=True
 DJANGO_SETTINGS_MODULE=aml.development_settings
-REACT_APP_API_ROOT=http://localhost:8000
-REACT_APP_EXPERIMENT_SLUG=fe
-REACT_APP_AML_HOME=https://www.amsterdammusiclab.nl
-REACT_APP_HTML_PAGE_TITLE=Amsterdam Music Lab Experiment
-REACT_APP_HTML_OG_DESCRIPTION=Listening experiments from the Amsterdam Music Lab. Test your musical knowledge and skills in engaging citizen-science experiments.
-REACT_APP_HTML_OG_IMAGE=https://app.amsterdammusiclab.nl/images/og-hooked.jpg
-REACT_APP_HTML_OG_TITLE=Amsterdam Music Lab Experiment
-REACT_APP_HTML_OG_URL=https://app.amsterdammusiclab.nl
-REACT_APP_HTML_BODY_CLASS=
\ No newline at end of file
+
+VITE_STRICT=false
+VITE_API_ROOT=http://localhost:8000
+VITE_EXPERIMENT_SLUG=fe
+VITE_AML_HOME=https://www.amsterdammusiclab.nl
+VITE_HTML_PAGE_TITLE=Amsterdam Music Lab Experiment
+VITE_HTML_OG_DESCRIPTION=Listening experiments from the Amsterdam Music Lab. Test your musical knowledge and skills in engaging citizen-science experiments.
+VITE_HTML_OG_IMAGE=https://app.amsterdammusiclab.nl/images/og-hooked.jpg
+VITE_HTML_OG_TITLE=Amsterdam Music Lab Experiment
+VITE_HTML_OG_URL=https://app.amsterdammusiclab.nl
+VITE_HTML_BODY_CLASS=
```

Comment:
I think renaming all these variables to `VITE` makes the purpose of them less clear. The `REACT_APP` here does not refer so much to the compilation procedure as to the role of the variables, which are about controlling the behaviour of the REACT frontend. Perhaps we could change these to `CLIENT` or `FRONTEND` instead, so that they are more descriptive.

Reply:
> I think renaming all these variables to `VITE` makes the purpose of them less clear. The `REACT_APP` here does not refer so much to the compilation procedure as to the role of the variables, which are about controlling the behaviour of the REACT frontend. Perhaps we could change these to `CLIENT` or `FRONTEND` instead, so that they are more descriptive.

Actually, the `REACT_APP` prefix is a [requirement for adding environment variables](https://create-react-app.dev/docs/adding-custom-environment-variables/) when using CRA (Create React App). [The same thing is true for VITE](https://vitejs.dev/guide/env-and-mode#env-files).

A possible way to avoid this is to use the variable names we want to use in the `.env` file, and then have the `docker-compose.yaml` file map them like this:

```
# .env file

CLIENT_API_ROOT=https://example.com
```

```
# docker-compose.yaml

client:
        environment:
          - VITE_API_ROOT=${CLIENT_API_ROOT}
```

Would you prefer such a solution or just use `VITE_`?